discretize/CurvilinearMesh.py

```diff
@@ -41,28 +115,48 @@ class CurvilinearMesh(BaseRectangularMesh, DiffOperators, InnerProducts, CurviVi
 
     _meshType = 'Curv'
 
-    def __init__(self, nodes):
-        assert type(nodes) == list, ("'nodes' variable must be a list of "
-                                     "np.ndarray")
-        assert len(nodes) > 1, "len(node) must be greater than 1"
+    nodes = properties.List(
+        "List of arrays describing the node locations",
+        Array(
+            "node locations in a dimension",
+            shape='*'
+        ),
+    )
 
-        for i, nodes_i in enumerate(nodes):
-            assert isinstance(nodes_i, np.ndarray), ("nodes[{0:d}] is not a"
-                                                     "numpy array.".format(i))
-            assert nodes_i.shape == nodes[0].shape, ("nodes[{0:d}] is not the "
-                                                     "same shape as nodes[0]"
-                                                     .format(i))
+    def __init__(self, nodes=nodes, **kwargs):
```

Comment:
Looks like you are using `nodes`, the `properties.List`, as the default value...? Seems fishy... though I may be missing something.

Reply:
thanks, this is a typo.